Driver/Driver/Util/UserDefaultsManager.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  UserDefaultsManager.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 17.05.2023.
+//
+
+import Foundation
+
+struct UserDefaultsManager {
```

Comment:
access token should be store in keychain not user defaults. It becomes the security problem while anybody reverse engineering.